samples/core/src/main/java/io/oasp/gastronomy/restaurant/general/configuration/BaseWebSecurityConfig.java

```diff
@@ -78,9 +82,9 @@ public void configure(HttpSecurity http) throws Exception {
 
     http
         //
-        .authenticationProvider(this.authenticationProvider)
         // define all urls that are not to be secured
-        .authorizeRequests().antMatchers(unsecuredResources).permitAll().anyRequest().authenticated().and()
+        .userDetailsService(userDetailsService()).userDetailsService(inMemoryUserDetailsManager()).authorizeRequests()
```

Comment:
so we configure two concurrent user details services here or does the second override the first? Looks strange. Needs at least some comment with explanation if that is really sane.